src/database/tests/integration/repositories/company.repository.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { Types } from 'mongoose';
+import { AppModule } from '../../../../app.module';
+import { DatabaseModule } from '../../../database.module';
+import { nanoid } from '../../../functions/nano-id.function';
+import { CompanyRepository } from '../../../repositories/company.repository';
+import { Company } from '../../../schemas/company.schema';
+
+const COMPANY_ID: string = nanoid();
+const COMPANY_MONGO_ID: Types.ObjectId = new Types.ObjectId();
+
+describe('CompanyRepository', () => {
+    let company: Company = <Company>{
+        _id: COMPANY_MONGO_ID,
+        id: COMPANY_ID,
+        name: 'Test',
+        email: 'Test.email@email.com',
+        phoneNumber: '123456789',
```

Comment:
thought(non-blocking): Consider using a dedicated library to create this kind of data like email and phone numbers to do more realistic tests.

Reply:
use faker for some fields now